Session/Shared/SessionTableViewModel.swift

```diff
@@ -11,6 +11,7 @@ import SessionUtilitiesKit
 protocol SessionTableViewModel: AnyObject, SectionedTableData {
     var dependencies: Dependencies { get }
     
+    var searchable: Bool { get }
```

Comment:
Just wondering whether we can add an extension to `SessionTableView` with a default `var searchable: Bool { false }` so that we don't need to add `let searchable: Bool = false` to all the different view models?